src/main/kotlin/nl/connectplay/scoreplay/models/dto/game/GameDto.kt

```diff
@@ -17,5 +18,5 @@ data class GameDto(
 
 // simple mapper
 fun Game.toDto() = GameDto(
-    id, name, description, publisher, minPlayers, maxPlayers, duration, minAge, releaseDate
+    id, null, name, description, publisher, minPlayers, maxPlayers, duration, minAge, releaseDate
```

Comment:
```suggestion
    id, scoringMethodId, name, description, publisher, minPlayers, maxPlayers, duration, minAge, releaseDate
```

tevens hoeft het geen extension functie te zijn, je kan dit gewoon in de Game klasse stoppen.

Reply:
the uitwerking van de scoringmethods zit niet in deze pull request